src/js/visual/configurator-prc.js

```diff
@@ -428,8 +455,10 @@ ripe.ConfiguratorPrc.prototype.changeFrame = async function(frame, options = {})
     }
 
     // in case the safe mode is enabled and the current configuration is
-    // still under the preloading situation the change frame is ignored
+    // still under the preloading situation the change frame is saved in
+    // and will be executed after the preloading
     if (safe && this.element.classList.contains("preloading")) {
+        this.pendingOperations = [["changeFrame", frame, options]];

```

Comment:
Maybe use a named tuple (object) instead of an anonymous one.
```suggestion
        this.pendingOperations = [{ operation: "changeFrame", "args": [frame, options] }];
```